contracts/system_addresses.go

```diff
@@ -30,4 +30,6 @@ var (
 	ConsolePrecompile = types.StringToAddress("0x000000000000000000636F6e736F6c652e6c6f67")
 	// AllowListContractsAddr is the address of the contract deployer allow list
 	AllowListContractsAddr = types.StringToAddress("0x0200000000000000000000000000000000000000")
+	// AllowListTransactionsAddr is the address of the transactions allow list
+	AllowListTransactionsAddr = types.StringToAddress("0x0200000000000000000000000000000000000002")
```

Comment:
Just curious, why we haven't taken `0x0200000000000000000000000000000000000001` for address? This way we have a gap in addresses.
```suggestion
	AllowListTransactionsAddr = types.StringToAddress("0x0200000000000000000000000000000000000001")
```

Reply:
Ok now I see, we are following the Avalanche specification, it is described in the JIRA task itself.